spring-cloud-dataflow-shell-core/src/test/java/org/springframework/cloud/dataflow/shell/command/TaskCommandTemplate.java

```diff
@@ -103,6 +107,65 @@ public long launchWithAlternateCTR(String taskName, String ctrAppName) {
 		return value;
 	}
 
+	/**
+	 * Launch a task and validate the result from shell on default platform.
+	 *
+	 * @param taskName the name of the task
+	 */
+	public String getTaskExecutionLog(String taskName) throws Exception{
+		long id = launchTaskExecutionForLog(taskName);
+		CommandResult cr = shell.executeCommand("task execution log --id " + id);
+		assertTrue(cr.toString().contains("Starting TimestampTaskApplication"));
```

Comment:
Checking `TimestampTaskApplication` here may not be correct as this method looks generic.